docs/content/testing/running-tests.md

```diff
@@ -0,0 +1,62 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+-----
+
+# Running tests
+
+Run the following commands in your terminal from the root of the repository.
+
+For the first run, you need these commands:
+
+```bash
+./prepare-workspace.sh
+./ci/build-test.sh
+```
+
+If you have modified anything in the code generator or docs linter tool;
+ to re-test these you need to re-run this command:
+
+```bash
+./ci/build-test.sh
+```
+
+If you have modified anything in the Unity Project; to re-test this you can
+ re-run this command:
+
+```bash
+./ci/test.sh
+```
+
+## Unity tests
+
+For Unity tests, you can also run tests from the Unity Test Runner Window.
+
+### 1. Open the Unity project in Unity 
+
+The unity project can be found in this directory within the repository: `/worker/unity`
+
+### 2. Open test runner window
+
+From the toolbar at the top of the Unity Editor, select "Window" > "Test Runner".
+
+### 3. Run tests
+
+You can press the "run all" button to run all tests. These will include the
```

Comment:
I'll go with your suggestion, it looks better

Reply:
* See note above about capitalisation of "run all"  to match what's on screen.
* Microsoft Manual of Style, which we use, says to use 'Select' rather than 'Press'. (https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click)
* No need to say "These will include..." as it's a statement of consistent fact, so "These include..."

Reply:
STOP PRESS on formatting, we are going with:

**run all** . <- bold.

So;
"To run all the tests, select **run all** . These include..."
(Assumes capitalisation of "run all" matches what's on screen.)